mem-01-b.xml

```diff
@@ -7371,7 +7463,37 @@ DSC {pIqaD:n:nolink} Schriftart: {Quvar:n:name,nolink} (Lieven L. Litaer)
 Ein besonderer Dank richtet sich an Marc Okrand ({marq 'oqranD:n:name}) für das Erschaffen der klingonischen Sprache.</column>
       <column name="notes_fa"></column>
       <column name="notes_sv"></column>
-      <column name="notes_ru"></column>
+      <column name="notes_ru">
+          {tlhIngan Hol boQwI':n:nolink} "Помощник по Клингонскому языку" - database version [[VERSION]].
+
+          Чтобы начать поиски, просто нажмите кнопку "Поиск" (увеличительное стекло) и печатайте в строке поиска
+
+          Чтобы услышать звуки на Клингонском языке, вам необходимо установить {Text-To-Speech engine:url:https://play.google.com/store/apps/details?id=org.tlhInganHol.android.klingonttsengine}
+
+          Рекомендуется установить {Klingon Keyboard for ASK:url:https://play.google.com/store/apps/details?id=com.anysoftkeyboard.languagepack.tlhIngan}. Тем не менее, чтобы облегчить процесс печати текста на клингонском на мобильной клавиатуре, стеганография (под названием "xifan hol")  может быть активирована в меню Preferences:
+
+          c ▶ {ch:sen:nolink} / d ▶ {D:sen:nolink} / f ▶ {ng:sen:nolink} / g ▶ {gh:sen:nolink} / h ▶ {H:sen:nolink} /
+          i ▶ {I:sen:nolink} / k ▶ {Q:sen:nolink} / s ▶ {S:sen:nolink} / x ▶ {tlh:sen:nolink} / z ▶ {':sen:nolink}
+          It is also possible to choose the alternate keymapping:
+          k ▶ {q:sen:nolink} / q ▶ {Q:sen:nolink}
+
+          Если вы столкнетесь с какими-либо проблемами, или у вас появятся предложения, пожалуйста отправьте сообщение на {email at De.vID.jonpIn@gmail.com:url:mailto:De.vID.jonpIn@gmail.com?subject=boQwI%27%20feedback&amp;body=Device:%20%0D%0AAndroid%20version:%20%0D%0A}
+
+          Поддержите Клингонский язык, купив {The Klingon Dictionary:src}, {Klingon for the Galactic Traveler:src}, {The Klingon Way:src}, {Conversational Klingon:src}, {Power Klingon:src}, и другую продукцию, связанную со Стар Трэком и Клингонами , от Pocket Books, Simon &amp; Schuster, and Paramount/Viacom/CBS Entertainment.
+
+          Klingon, Star Trek, and related marks are trademarks of CBS Studios, Inc., and are used under "fair use" guidelines.
```

Comment:
The following line seems to be the translation, so this line should be deleted?

Reply:
Yesd.I will delete it

Reply:
Can you rebase from master? As I've explained, I've already made the changes, so you don't have to do anything.